src/Resources/public/Admin.js

```diff
@@ -340,10 +340,20 @@ var Admin = {
 
             // Keep only changed values
             $form.find('[name*=filter]').each(function (i, field) {
-                if (JSON.stringify(defaults[field.name] || '') === JSON.stringify($(field).val())) {
+                var defaultValue = field.multiple ? [] : '';
+
+                if (JSON.stringify(defaults[field.name] || defaultValue) === JSON.stringify($(field).val() || defaultValue)) {
                     field.removeAttribute('name');
                 }
             });
+
+            // Simulate a reset if no value is different from the default ones.
+            if ($form.find('[name*=filter]').length === 0) {
+                var field = $(document.createElement("input"));
+                field.attr("name", 'filters').attr("type","hidden");
```

Comment:
Just for consistency:
```suggestion
                field.attr('name', 'filters').attr('type', 'hidden');
```

Reply:
Sure, done